src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities/TraceUtilities.cs

```diff
@@ -11,6 +11,30 @@
 
 namespace Microsoft.Diagnostics.Monitoring.WebApi
 {
+    internal sealed class GcCollectConfiguration : MonitoringSourceConfiguration
```

Comment:
This will be removed after https://github.com/dotnet/diagnostics/pull/4568 correct?

Reply:
The PR https://github.com/dotnet/diagnostics/pull/4568 is merged, but I still can't remove this class just yet. It appears that we will need to publish a new nuget package there that contains the new class before this can be removed.